transform-plugins/docs/Compressor-transform.md

```diff
@@ -3,26 +3,27 @@
 
 Description
 -----------
-Compresses configured fields. Multiple fields can be specified to be compressed using different compression algorithms.
-Plugin supports SNAPPY, ZIP, and GZIP types of compression of fields.
+Compressor transform is used for compressing the fields of a dataset using the configured compression algorithm. You can specify multiple fields to be compressed using different compression algorithms.
+The accelerator supports SNAPPY, ZIP, and GZIP types of compression.
 
 
 Configuration
 -------------
-**compressor:** List of key value pairs. Key represents the input field that needs to be compressed and Value represents the compression algorithm to use.
+You need to specify the following configuration for the accelerator:
 
-**Output Schema:** Output schema will be same as input schema except fields in output schema that are compressed will have the same field name 
-but they will be of type *Non-nullable* ``BYTES``.
```

Comment:
Add getSchema property here as well. which is responsible to get schema on the basis of compressor field and apply that schema in output schema  section . User do not need to change compressor fields manually to bytes in out schema .